visualizer/index.html

```diff
@@ -253,6 +253,7 @@
       padding: 0;
     }
     .contextMenu li {
+      position: relative;
```

Comment:
Ah, got it. Maybe leave a comment here then? Like `/* For position submenus */`.